ulfhara.lic

```diff
@@ -122,6 +123,23 @@ class UlfHara
         combo
       when /help any of our wounded you encounter/
         fput("accept task")
+        if DRStats.empath? && !DRSpells.active_spells.include?('Regenerate') && @settings.waggle_sets['healing']['Regenerate']
+          data = @settings.waggle_sets['healing']['Regenerate']
```

Comment:
You will be surprised at how many folks have 'regenerate' vs 'Regenerate'. This isn't a call to action, but a note that at some point the keys in the Hash should be downcased before comparing.